### ISSUE TITLE
Unable to Coerce return values when they are Protos

### ISSUE THREAD

**Issue by kmaphoenix** [google-gemini/cookbook#78]
# TLDR
When calling a function that returns a proto (or list of protos), a ValueError is thrown because it cannot coerce the results.

![image](https://github.com/google-gemini/cookbook/assets/13949241/0e4034a0-c28f-4b65-a62b-7fbdf0d38635)

# Potential Fix
I'm assuming the result from the function call needs to be in some standard python type (i.e. list, str, int, etc.)
Instead of attempting to coerce each field in the proto, can we simply return the entire proto as a string?

If you see here, I've written a wrapper method around the original, where I simply take each proto and coerce to str.
The function call is able to then work properly with the string format.

![image](https://github.com/google-gemini/cookbook/assets/13949241/5f316867-9d26-40eb-801c-c68fe8d4c35e)
